### ISSUE TITLE
Add CONTRIBUTING.md

### ISSUE THREAD

**Issue by stuaxo** [Libvisual/libvisual#185]
CONTRIBUTING.md and possibly an issue template could be good.

I had a really brief look at a few and most are much longer than needed for a project with this few contributors - the best way might be to look at examples from other projects then write our own based on the bits we like.

To start with we can collect examples here that we like.

A lot of things in other CONTRIBUTING.md probably aren't needed on this project (yet), just because of the small amount of developers.   The document should aim to streamline getting new developers started.

**Comment by stuaxo** [comment]
Example:  The Atom contributing.md has some good sections and I like that they surfaced "I don't want to read this whole thing, I just have a question!!!"

https://github.com/atom/atom/blob/master/CONTRIBUTING.md

**Comment by kaixiong** [comment]
Yes we should do this. There is actually a hastily written wiki page titled [Hacking Guide](https://github.com/Libvisual/libvisual/wiki/Hacking-guide) which we can source from.
